lib/tasks/post_precompile_assets.rake

```diff
@@ -0,0 +1,6 @@
+task after_assets_precompile: :environment do
+  # Runs postcss/autoprefixer to ensure that our CSS works for all the browsers we support.
+  system("yarn postcss")
+end
+
+Rake::Task["assets:precompile"].enhance ["after_assets_precompile"]
```

Comment:
If you want `after_assets_precompile` to run AFTER `assets:precompile` you will need to use the block notation here. As is this will run the `after_assets_precompile` BEFORE `assets:precompile`. How do I know? [I already made that mistake not too long ago](https://dev.to/molly_struve/rake-task-enhance-method-explained-3bo0) 😉